distros/debian/control

```diff
@@ -3,17 +3,16 @@ Section: net
 Priority: optional
 Maintainer: Isaac Connor <iconnor@connortechnology.com>
 Build-Depends: debhelper (>= 9), cmake
-    , libphp-serialization-perl
```

Comment:
Why are you dropping this dependency?

Reply:
Oops, that was a mistake. I rebased and squashed this fixup in.